src/adapters/unruly.js

```diff
@@ -0,0 +1,108 @@
+import { ajax } from 'src/ajax'
+import bidfactory from 'src/bidfactory'
+import bidmanager from 'src/bidmanager'
+import * as utils from 'src/utils'
+import { STATUS } from 'src/constants'
+import { Renderer } from 'src/Renderer'
+
+function createRenderHandler({ bidResponseBid, rendererConfig }) {
+  function createApi() {
+    parent.window.unruly.native.prebid = parent.window.unruly.native.prebid || {}
```

Comment:
In discussion it was mentioned that there is no intention of breaking out of an iframe here, in which case `parent` shouldn't be needed.